index.php

```diff
@@ -0,0 +1,47 @@
+
+<html lang="en">
+    <head>
+        <link rel="stylesheet" type="text/css" href="normalize.css">
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
+        <title>My Collection</title>
+    </head>
+    <body>
+        <h1>My Handbag Collection</h1>
+        <nav>
+            <div class="navbar">
+                <a class="underline">Collection</a>
+                <a class="underline">Add Item</a>
+                <a class="underline">Edit Item</a>
+            </div>
+        </nav>
+    <main>
+        <?php
+        require_once('getDbFunction.php');
+        $db=getDb();
+        $query=$db->prepare("SELECT `brand`,`image`,`model`,`price`,`materials` FROM `bags`;");
+        $query->execute();
+        $result = $query->fetchAll();
```

Comment:
Can this be moved into it's own function? Mainly for separation of concerns. So rather than having this query run directly in index.php, make a getBags (or whatever you wanna call it) function in a different file that you can call here instead. Helps keep things a bit tidier.